core/server/primarykinds.go

```diff
@@ -25,7 +25,7 @@ const (
 	// It's multiplied with the digit following the "beta" suffix.
 	// Beta versions are more stable than alpha versions but might still contain bugs.
 	// They are given a higher weight than alpha versions.
-	betaWeight = 500
+	betaWeight = 50
```

Comment:
Just for clarification: we determine which kinds to use at runtime?

Reply:
yes they are resolved at startup here https://github.com/weaveworks/weave-gitops/blob/17e51daa99e6f5f6d1732b8b6e1064d8d67b8fef/core/server/primarykinds.go#L126

Reply:
Seems like a bit of an anti-pattern...